### ISSUE TITLE
[COGNITION]: GIBCT Section 116 - Estimated benefits should be calculated on button press, not form control change

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#3536]
## Description

There are multiple controls affecting the Your Estimated Benefits calculated values. These controls should be consolidated into one area and include a Submit button to calculate values one time and manage focus for better user cognition. Screenshot attached below.

Please review any design changes with the VSP design team for user flow and 508 considerations.

/cc @emilywaggoner && @CrystabelReiter 

## Point of Contact

**VFS Point of Contact:** _Emily, Crystabel, Trevor_

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/1276

## Acceptance Criteria

* As a user, I want to be able to select a school, enter my tuition costs, and scholarships, then press a submit button to have my benefit values calculated. Depending on my screen size, I may not see the benefits calculated values changing right now because they're off-screen.
* As a screen reader user, I want to hear my estimated benefits have been updated. This can be accomplished by listening for the Submit button click event and moving focus to the appropriate heading.

## WCAG or Vendor Guidance (optional)

* [On Input: Understanding SC 3.2.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-unpredictable-change.html)

## Screenshots or Trace Logs

![Screen Shot 2019-11-18 at 11.37.39 AM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/605a68e3-3d62-4b7c-b000-ce7298be778f)

**Comment by WillMcCormack** [comment]
I sent an email over regarding this defect yesterday (11/18). The team believes this to be more of an UI/UX design consideration than a 508 defect.

**Comment by KevinMHoffmanUSDS** [comment]
@WillMcCormack Could you drop the copy of that e-mail into this thread? Thanks!

**Comment by desireeturner** [comment]
@1Copenut @KevinMHoffmanUSDS @WillMcCormack Is there a path forward identified on this issue?

**Comment by KevinMHoffmanUSDS** [comment]
Hello, @desireeturner and @WillMcCormack - long time no see! It's the VA's position that based on https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/F36 this is indeed a 508 failure and not a UI/UX design consideration. 

Let us know if the guidance provided is sufficient for revisions, or you'd like to discuss further. Cheers! 

cc @mattpointzxer0

**Comment by desireeturner** [comment]
@KevinMHoffmanUSDS We have this on our radar and in our backlog. We will pick it up as soon as we have the capacity.

**Comment by desireeturner** [comment]
@KevinMHoffmanUSDS @1Copenut This work has been completed as part of our initiative to update the "Estimate your benefits" section of the Comparison Tool. These changes went live on 6/16 on Production. Please let us know if you need anything additional to close this out. Thank you!

**Comment by 1Copenut** [comment]
@desireeturner I am good with closing this issue. The new workflow to let users toggle settings and update benefits calculations inside each accordion is clear and with the last tweaks being done, will work well across all screen readers.

/cc @KevinMHoffmanUSDS for awareness.

**Comment by desireeturner** [comment]
Great... thank you! Closing now.
